_posts/2020-12-07-autumn-2020-newsletter.md

```diff
@@ -87,7 +87,7 @@ In November, [Alba](https://publiccode.net/team/alba-roza.html) gave a presentat
 
 [Felix](https://publiccode.net/team/felix-faassen.html) wrote [an article about how government and market parties are jointly building the OpenZaak community](https://opensource.pleio.nl/groups/view/57979222/open-source/blog/view/57979368/gemeenten-en-de-markt-bouwen-samen-de-openzaak-community) for the Dutch national open source forum ([opensource.pleio.nl](https://opensource.pleio.nl/)).
 
-Last week, [Jan](https://publiccode.net/team/jan-ainali.html) ran a workshop on how to effectively do "Open Source in the Public Sector" at the [Swedish GovTechDay](https://www.govtechday.se/program-2020#block-yui_3_17_2_1_1603099025104_121820) ([slides](https://hackmd.io/@Ainali/H1EQUI99D#/) in Swedish).
+Last week, [Jan](https://publiccode.net/team/jan-ainali.html) ran a workshop on how to effectively do "Open Source in the Public Sector" at the [Swedish GovTechDay](https://www.govtechday.se/) ([slides](https://hackmd.io/@Ainali/H1EQUI99D#/) in Swedish).
```

Comment:
I believe we can remove the govtechday link completely, since it redirects to a new site with no trace of the event. (also in _posts/2020-12-14-notes-from-community-call-3-december-2020.md  and  script/test.sh )

Reply:
okay, hmm. I suggest we do that as an add-on commit.

Reply:
Yes, this link could be used instead: https://web.archive.org/web/20201105140359/https://www.govtechday.se/program-2020